statistics-rest/src/test/java/org/uniprot/api/statistics/mapper/StatisticMapperTest.java

```diff
@@ -0,0 +1,96 @@
+package org.uniprot.api.statistics.mapper;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.uniprot.api.common.repository.search.facet.FacetProperty;
+import org.uniprot.api.statistics.StatisticAttributeConfig;
+import org.uniprot.api.statistics.entity.EntryType;
+import org.uniprot.api.statistics.entity.UniprotkbStatisticsEntry;
+import org.uniprot.api.statistics.model.StatisticsModuleStatisticAttribute;
+import org.uniprot.api.statistics.model.StatisticModuleStatisticType;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.mockito.Mockito.when;
+import static org.uniprot.api.statistics.TestEntityGeneratorUtil.STATISTICS_CATEGORIES;
+import static org.uniprot.api.statistics.TestEntityGeneratorUtil.STATISTICS_ENTRIES;
+
+@ExtendWith(MockitoExtension.class)
+class StatisticMapperTest {
+    private static final Map<String, FacetProperty> FACET_MAP = new HashMap<>();
+    public static final String LABEL_0 = "label0";
+    @Mock
```

Comment:
label0, cat0 etc can be made as String constant at the common place e.g. TestEntityGeneratorUtil and then used everywhere.